e3db/types/meta.py

```diff
@@ -25,7 +25,7 @@ def __init__(self, json):
         self.__writer_id = uuid.UUID(json['writer_id'])
         self.__user_id = uuid.UUID(json['user_id'])
         self.__record_type = str(json['type'])
-        self.__plain = json['plain'] if 'plain' in json else None
+        self.__plain = json['plain'] if 'plain' in json else {}
```

Comment:
Is this a necessary change to fix a specific bug or something else we're introducing? Feels odd to sneak it in with an "Add testing setup" merge ...

Reply:
its a bug fix - we decided that plain meta should always be empty object at some point. See https://toznysecurity.atlassian.net/browse/E3DB-757.